builtin.c

```diff
@@ -474,6 +474,25 @@ XS(XS_builtin_indexed)
     XSRETURN(retcount);
 }
 
+XS(XS_builtin_load_module);
+XS(XS_builtin_load_module)
+{
+    dXSARGS;
+
+    if (items != 1)
+        croak_xs_usage(cv, "arg");
+    SV *st0 = ST(0);
+    SV *module_name = newSVsv(st0);
+    ENTER;
+    lex_start(NULL, NULL, 0);
+    I32 floor = start_subparse(FALSE, 0);
+    OP *op = newSVOP(OP_CONST, 0, module_name);
+    op->op_private |= OPpCONST_BARE;
+    utilize(0, floor, NULL, op, NULL);
+    LEAVE_SCOPE(floor);
+    LEAVE;
```

Comment:
My apologies - I think I sent you down the wrong path in my original email.

Having looked more into this since then I've remembered we have a `load_module` function at the C level as well. That would be a far simpler way to do all of this.

```c
load_module(PERL_LOADMOD_NOIMPORT, module_name, NULL, NULL);
```

This is the one I was thinking of when I mentioned `utilize()`. Sorry for the misdirection.